navigation/ca_move_base/launch/move_base_flex.launch

```diff
@@ -0,0 +1,57 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+  <arg name="tf_prefix" value="$(arg ns)_tf"          doc="Tf prefix"/>
+
+  <arg name="laser" default="$(optenv LASER rplidar)" doc="Laser for mapping the environment"/>
+
+  <arg name="global_planner" value="$(optenv GLOBAL_PLANNER gbp)"/>
```

Comment:
Use the same value in `move_base` too.

Reply:
Done!

Reply:
Since you told me the other day that `gbp` is newer (and better?) than `navfn` consider using it as the default option!

Reply:
Done!